CMakeLists.txt

```diff
@@ -177,8 +177,9 @@ find_package(Boost REQUIRED COMPONENTS
 	iostreams
 	filesystem
 	date_time
-	system
 	program_options
+	OPTIONAL_COMPONENTS
```

Comment:
Made PR a draft as need to figure out CMake < 3.11 where `OPTIONAL_COMPONENTS` doesn't exist.